website/docs/en/index.md

```diff
@@ -30,7 +30,7 @@ features:
   - title: Webpack Compatible
     details: Interoperates with existing webpack plugins and config. No need to reinvent your ecosystem from scratch.
     icon: 📦
-    link: /guide/compatibility/loader
+    link: /guide/compatibility/config-diff
```

Comment:
link  to `/guide/compatibility/plugin` ?

Reply:
done.